### PR TITLE
🌠 added a custom_loadingoverlay example

### PR DESCRIPTION
**Pull request by pip-install-python**
example_video_editor

added an example for a custom dmc.LoadingOverlay

### CODE REVIEW

**Comment by AnnMarieW** [comment]
Hi @pip-install-python 

Thanks for the PR!  This will be a nice addition :slightly_smiling_face: 

To promote dmc more, it would be cool if you used the `dmc.Image` component instead of `html.Img`  Also would you like to use one of the dmc charts?   You could use the sample data shown in the docs, then we wouldn't need to add the numpy and dependency.

**Comment by pip-install-python** [comment]
I made those changes, using dmc.Image and used a dmc chart, also made the loading .gif more cute 🐌

**Comment by pip-install-python** [comment]
Screenshot 2024-10-04 at 12 04 48 PM
Feel like we could make this section in the Keyword Arguments a bit more descriptive to show the 4 options available in the dictionary, wasn't sure how to go about it though. 

Like how to edit the .. kwargs::LoadingOverlay

**Comment by pip-install-python** [comment]
Screenshot 2024-10-04 at 12 07 26 PM
Screenshot 2024-10-04 at 12 07 46 PM

also figure it was worth mentioning, it seems like all LoadingOverlays clip into the navbar when scrolled to a certain point, very small bug but figured I'd mention and show some examples.

**Review by AnnMarieW — COMMENTED** [review]
Love the gif - very cute :slightly_smiling_face:

File: docs/loadingoverlay/customimageoverlay.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
+import time
```
Comment (AnnMarieW) [inline docs/loadingoverlay/customimageoverlay.py:1]: can delete this import

File: docs/loadingoverlay/customimageoverlay.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+        "Tablets": random.randint(20, 150),
+    }
+    for month in ["January", "February", "March", "April", "May", "June"]
+]
+
+component = dmc.Box(
+    children=[
+        dmc.Stack(
+            pos="relative",
+            p=5,
```
[the remark this answers is not in the document]
Comment (pip-install-python) [reply docs/loadingoverlay/customimageoverlay.py]: Not sure why the snail.gif was cut off for you, tested this on a bunch of different display sizes and mobile / tablet and it seems to be rendering correctly for me. Changed padding to 10

**Comment by AnnMarieW** [comment]
nice catch catch on the duplicate id!

File: docs/loadingoverlay/customimageoverlay.py [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+component = dmc.Box(
+    children=[
+        dmc.Stack(
+            pos="relative",
+            p=10,
+            children=[
+                dmc.LoadingOverlay(
+                    visible=True,
+                    id="custom-loading-overlay",
+                    style={'zIndex': 10},
```
Comment (AnnMarieW) [inline docs/loadingoverlay/customimageoverlay.py:25]: The `zIndex`is a prop for this component, so you can just do it like this:

 `zIndex=10`

File: docs/loadingoverlay/customimageoverlay.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (AnnMarieW) [reply docs/loadingoverlay/customimageoverlay.py]: Ok - thanks.  I should be fine and if not, it will be easy to change later :-)

**Comment by AnnMarieW** [comment]
Awesome!  Thanks!
# :woman_dancing:

**Review by AnnMarieW — APPROVED** [review]

**Pull request merged**